sources/web/build.sh

```diff
@@ -32,7 +44,12 @@ mkdir -p $WEB_DIR
 # Experimental UI build step
 cd datalab/polymer
 npm run build
-rsync -avpq ./build/experimental/ ../static/experimental
+if [[ $DEBUG == 1 ]]; then
+  rsync -avpq ./build/polymer_unbundled/ ../static/experimental
+else
+echo "Using bundled polymer resources.."
+  rsync -avpq ./build/polymer_bundled/ ../static/experimental
```

Comment:
Just to double check, this means that everything is still getting placed into /datalab/web/static/experimental, right?  We have some references to that directory, so let me know if that is not true.

Reply:
Yup, this doesn't change the end result, just the intermediate copy step.